lib/npm-check-updates.js

```diff
@@ -97,20 +97,54 @@ function createDependencyTable() {
             middle: ''
         }
     });
+    return baseTable;
+}
+
+/**
+ * @param {object} args
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function getOwnerPerDependency(args, options) {
+    const packageManager = vm.getPackageManager(options.packageManager);
+    args.owner = {};
+    await Promise.all(
+        Object.keys(args.to).map(async (dep) => {
+            const from = args.from[dep] || null;
+            const to = args.to[dep] || null;
+            const ownerChanged = await packageManager.packageAuthorChanged(dep, from, to);
+            // eslint-disable-next-line require-atomic-updates
+            args.owner[dep] = ownerChanged;
```

Comment:
Avoid mutation. Instead return a new object from each iteration of `map`.

e.g.
```js
return Object.assign({}, args, {
  owner: Object.assign({}, args.owner, {
    [dep]: ownerChanged
  }
})
```